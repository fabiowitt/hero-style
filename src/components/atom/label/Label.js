import React from 'react';
import PropTypes from 'prop-types';

const Label = ({ name, children }) => {
  return <label htmlFor={name}>{children}</label>;
};

Label.propTypes = {
  name: PropTypes.string,
  children: PropTypes.string.isRequired
};
Label.defaultProps = {
  name: null
};

export default Label;