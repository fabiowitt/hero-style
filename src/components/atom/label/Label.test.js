import React from 'react';
import renderer from 'react-test-renderer';
import Label from './Label';

describe('Label', () => {
  it('should match with the snapshot', () => {
    const wrapper = renderer
      .create(<Label name={'lbl'}>Text</Label>)
      .toJSON();
    expect(wrapper).toMatchSnapshot();
  });
});