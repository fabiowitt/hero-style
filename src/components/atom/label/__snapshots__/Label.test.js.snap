// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label should match with the snapshot 1`] = `
<label
  htmlFor="lbl"
>
  Text
</label>
`;
